dsi-iepg/cas-connection/0.1/config/packages/security.yaml

```diff
@@ -0,0 +1,16 @@
+security:
+    providers:
+        app_user_provider:
+            entity:
+            class: App\Entity\User
+            property: login
+
+    firewalls:
+        main:
+            provider: app_user_provider
+            custom_authenticator: Iepg\Bundle\Cas\Controller\CasAuthenticator
+
+    access_control:
+        - { path: ^/admin, roles: ROLE_ADMIN }
+        - { path: ^/cas-connection/cas-admin, roles: ROLE_ADMIN }
+        - { path: ^/cas-connection/cas-user, roles: ROLE_USER }
```

Comment:
This file should be removed as it will conflict with the one installed by the security component. Instead, you should add  post install file to explain the next step about how to configure the bundle.